jsonschema-hyperschema.xml

```diff
@@ -124,152 +129,235 @@
+                applications that expect plain JSON, while providing full hypermedia capabilities
+                for hyper-schema-aware applications and user agents.
             </t>
-            <figure>
-                <preamble>
-                    Here is an example JSON Schema defining hyperlinks:
-                </preamble>
-                <artwork>
-<![CDATA[
-{
-    "title": "Written Article",
-    "type": "object",
-    "properties": {
-        "id": {
-            "title": "Article Identifier",
-            "type": "number",
-            "readOnly": true
-        },
-        "title": {
-            "title": "Article Title",
-            "type": "string"
-        },
-        "authorId": {
-            "type": "integer"
-        }
-    },
-    "required" : ["id", "title", "authorId"],
-    "links": [
-        {
-            "rel": "self",
-            "href": "/article{?id}"
-        },
-        {
-            "rel": "author",
-            "href": "/user?id={authorId}"
-        }
-    ]
-}
-]]>
-                </artwork>
-                <postamble>
-                    This example schema defines the properties of the instance as well as link
-                    relations for the instance, with URIs incorporating values from the instance.
-                </postamble>
-            </figure>
+            <t>
+                Client-side user agents can detect the presence of hyper-schema by looking for
+                the application/schema+json media type and a "$schema" value that indicates the
+                presence of the hyper-schema vocabulary.  A user agent can then use an
+                implementation of JSON Hyper-Schema to provide an interface to the
+                combination of the schema and instance documents as a single logical
+                representation of a resource, just as with any single-document hypermedia
+                representation format.
+            </t>
+            <t>
+                Hyper-schemas allow representations to take up fewer bytes on the wire, and
+                distribute the burden of link construction from the server to each client.
+                A user agent need not construct a link unless a client application requests
+                that link.
+            </t>
+            <t>
+                JSON Hyper-Schema can also be used on the server side to generate other
+                link serializations at runtime, such as HTTP Link headers or
+                <xref target="I-D.kelly-json-hal">JSON HAL</xref>.  Because hyper-schema offers
+                a superset of the features of many other systems, using it on the server
+                can make it easier to support multiple representation media types and
+                linking mechanisms.  It is even possible to support features such as JSON HAL's
+                embedded resources by automatically following links on the server and
+                inlining the results.
```

Comment:
@dlax I can see how this would be a little confusing.

JSON Hyper-Schema is a separate document from the response (or request) document.  Instead of sending the links in the document, we use this separate document as instructions for building them.

To some people, that is a benefit.  The responses take up less space on the wire, and people who want to treat the system as a "traditional" non-HATEOAS HTTP API can easily do so.

To others, it is less desirable.  They want self-contained responses.  Additionally, given the variety of hypermedia-ish formats based on JSON, application developers may prefer various representations.  This paragraph is explaining how a JSON Hyper-Schema system can fulfill those use cases alongside of its traditional, client-side role.

When giving examples, I like to give at least two.  HTTP Link headers are one way to generate links server-side and send them in the response.  JSON HAL is a simple and at least somewhat popular format that would be easy to generate as well.  So you could offer both hyper-schema and HAL representations quite cheaply.  And unlike Siren or JSON API, HAL does not significantly constrain the representation structure.  It would be hard to use Siren or JSON API in this way, and impossible to do both as their structural restrictions are not compatible with each other.  So JSON HAL seemed the best 2nd example.

This paragraph is definitely not essential, but I do feel like it's an important use case for Hyper-Schema that is often overlooked.

Reply:
I understand all that and definitely agree it's worth mentioning as it highlights the nice "separation" of JSON Hyper-Schema. Just wondering if the introduction is the best place.
Maybe something to be moved to the web site, in particular since people may first read information there before getting into the spec. 

In general, about moving things from the spec to the web site, it's probably not a priority at this point and we can leave everything in the spec during the rewrite time while spotting "candidates for move".